experimental/plugins/auditlog_formatter_tinygo_test.go

```diff
@@ -0,0 +1,80 @@
+// Copyright 2022 Juan Pablo Tosso and the OWASP Coraza contributors
+// SPDX-License-Identifier: Apache-2.0
+
+//go:build tinygo
+// +build tinygo
+
+// Aimed to tinygo, initializing a dedicated serial writer
+package plugins_test
+
+import (
+	"fmt"
+	"io"
+
+	"github.com/corazawaf/coraza/v3"
+	"github.com/corazawaf/coraza/v3/experimental/plugins"
+	"github.com/corazawaf/coraza/v3/experimental/plugins/plugintypes"
+)
+
+type testFormatterTiny struct{}
```

Comment:
nit: do we need the Tiny suffix given that we have the build tags. Same for the wasm formatter.

Reply:
Yep, not really needed, removed, thanks